### PR TITLE
GET /data -> added some greeting by random names

### PR DESCRIPTION
**Pull request by tzavidas**
On the GET /data route, I added some greeting, which randomly chooses a name based on a predefined String [] array of names.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -20,13 +20,27 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import java.io.PrintWriter;
+import java.lang.Math.*;
```
Comment (maxwbuckley) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Only import what you use. Don't use *

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -20,13 +20,27 @@
...
+
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private static String[] nicknames = { "Nick", "nicktz1408", "Nikolaos", "an intern Googler", "a Noogler", "a STEPer" };
```
Comment (maxwbuckley) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: make final and change the name to all upper case

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (maxwbuckley) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (maxwbuckley) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

**Review by maxwbuckley — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (tzavidas) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Fixed! Actually, the math library is a single package, so I had just to remove the asterisk and it worked! My bad! :)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (tzavidas) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Done!

**Pull request merged**